crates/host/src/module.rs

```diff
@@ -139,38 +147,53 @@ pub const WASM_METERING_LIMIT: u64 = 100_000_000_000;
 /// We don't want tests to run forever, and it can take several minutes for 100 giga ops to run.
 pub const WASM_METERING_LIMIT: u64 = 10_000_000;
 
+#[cfg(feature = "wasmer_sys")]
 /// Generate an engine with a wasm compiler
 /// and Metering (use limits) in place.
-pub fn make_compiler_engine() -> Engine {
+pub fn make_engine() -> Engine {
```

Comment:
Renamed for accuracy, as this function will create a compiler engine or interpreter engine depending on the feature flag.